<javaelement>
	<package>
		javax.management.remote
	</package>
	<type>
		<name>
			public class JMXServerErrorException  extends IOException
		</name>
		<description>
			Exception thrown as the result of a remote MBeanServer method invocation when an Error is thrown while processing the invocation in the remote MBean server. A JMXServerErrorException instance contains the original Error that occurred as its cause.
		</description>
		<methods>
			<method>
				<name>
					public JMXServerErrorException(String s,                         Error err)
				</name>
				<description>
					Constructs a JMXServerErrorException with the specified detail message and nested error.
				</description>
				
			</method>
			<method>
				<name>
					public Throwable getCause()
				</name>
				<description>
					Description copied from class: Throwable Returns the cause of this throwable or null if the cause is nonexistent or unknown. (The cause is the throwable that caused this throwable to get thrown.) This implementation returns the cause that was supplied via one of the constructors requiring a Throwable, or that was set after creation with the Throwable.initCause(Throwable) method. While it is typically unnecessary to override this method, a subclass can override it to return a cause set by some other means. This is appropriate for a "legacy chained throwable" that predates the addition of chained exceptions to Throwable. Note that it is not necessary to override any of the PrintStackTrace methods, all of which invoke the getCause method to determine the cause of a throwable.
				</description>
				
			</method>
			
		</methods>
		
	</type>
	
</javaelement>
